lib/event.rb

```diff
@@ -0,0 +1,39 @@
+class Event
+  attr_reader :name, :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+    @ages.max
+  end
+
+  def min_age
+    @ages.min
+  end
+
+  def average_age
+    average = ages.sum / ages.length.to_f
```

Comment:
You don't really need to assign a variable here - it will work just as well without it!